<!DOCTYPE HTML>
<html>
  <head>
    <title>CELLS Workshop</title>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
    <link rel="stylesheet" href="assets/css/main.css?v=4" />
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.9/css/all.css" integrity="sha384-5SOiIsAziJl6AWe0HWRKTXlfcSHKmYV4RBF18PPJ173Kzn7jzMyFuTtk8JA7QQG1" crossorigin="anonymous">
    <link href="https://fonts.googleapis.com/css?family=Roboto:300,300italic,400,400italic,500,500italic,600,600italic,700,700italic,900,900italic" rel="stylesheet">
    <style>
      .mySlides {
        display:none;
      }
    </style>
  </head>

  <body>
    <center>

      <!-- Header -->
      <header id="header">

      	<nav id="nav">
      	  <ul>
            <li><a href="#page_disc">DISC'20</a></li>
      	    <li><a href="#page_scope">Scope</a></li>
      	    <li><a href="#page_program">Program</a></li>
            <li><a href="#page_local">Local</a></li>
            <li><a href="#page_speakers">Speakers</a></li>
      	    <li><a href="#page_commitee">Committee</a></li>
            <li><a href="#page_sponsors">Sponsors</a></li>
            <li><a href="#page_previous">Previous workshops</a></li>
            <li><a href="https://parsys.lri.fr/CELLS/2019.html">CELLS'19</a></li>
      	  </ul>
      	</nav>

      </header>

      <!-- Toggle Menu -->
      <a href="#menu" class="navPanelToggle">
      	<img src="logos/menu.svg"/>
      </a>
	
      <!-- Main part --> 
      <section id="main">
        
        <!-- main page -->
        <section class="page contentheight" id="first_page">
        

          <div class="bannerdiv">
           <img src="images/cells.svg" class="bannerimage">
          <span class="caption">CELLS: Workshop on Computing among Cells</span>
          <!--span class="caption2">October 15th and 16th, 2020 (online)<br><br></span>
          <a href="https://forms.gle/6WQeowDx46yH37Hi7" class="alt-reg">Register here</a><br><br><br>
          <a href="CELLS.ics" class="alt-button">Add events to calendar</a-->
          </div>

        </section>

        <section class="page contentheight" id="page_disc">

          <h1>DISC'20</h1>
          <div class="rightcol">
            <h1 class="alt">DISC'20</h1>
            <p>
            CELLS is a DISC 2020 workshop. Please see <a href="http://www.disc-conference.org/wp/disc2020/workshops/" class="alt">the DISC page</a> for further information.
            The workshop will take place online.
            Attendance to the workshop is free of charge.
            Zoom links to the talks are provided after registration.
            </p>
            <!--b>Free registration:</b>
            <p><a href="https://forms.gle/6WQeowDx46yH37Hi7" class="alt">Registration</a> is now open via the DISC 2020 page.
            </p-->
          </div>

        </section>

        <!-- page 2 -->
        <section class="page contentheight" id="page_scope">

          <h1>Scope</h1>
          <div class="rightcol">
          	<h1 class="alt">Scope</h1>
            <p>
            The workshop is intended to cover topics both from computer science and microbiology with a focus on computation emerging from microbiological units (bacteria, viruses, DNA, etc.) interacting with each other.
            </p>
            <p>
            Topics include, but are not limited to: synthetic biology, bacterial computing, DNA computing, population protocols, Markov chains, modeling with cellular automata, infection models, and evolutionary models.
            </p>
          </div>

        </section>


                <!-- page -->
        <section class="page contentheight" id="page_program">

          <h1>Program</h1>
          <div class="rightcol">
            <h1 class="alt">Program</h1>
            <p>
            Talks will be held online on October 15th and 16th, 2020 (starting afternoon, Central European Summer Time).
            </p>

            <p>
            Thursday, October 15, 2020 (CEST times):
            </p>
            <ul>
            <li>5:30 PM. &nbsp;&nbsp;&nbsp;<a href="#speaker-jeff_hasty" class="alt">Jeff Hasty</a></li>
            <li>6:30 PM. &nbsp;&nbsp;&nbsp;<a href="#speaker-david_doty" class="alt">David Doty</a></li>
            <li>7:30 PM. &nbsp;&nbsp;&nbsp;30min break</li>
            <li>8:00 PM. &nbsp;&nbsp;&nbsp;Discussion/Socializing</li>
            </ul>
            </p>

            <p>
            Friday, October 16, 2020 (CEST times):
            </p>
            <ul>
            <li>2:00 PM. &nbsp;&nbsp;&nbsp;<a href="#speaker-kurt_mehlhorn" class="alt">Kurt Mehlhorn</a></li>
            <li>3:00 PM. &nbsp;&nbsp;&nbsp;<a href="#speaker-tom_de_greef" class="alt">Tom de Greef</a></li>
            <li>4:00 PM. &nbsp;&nbsp;&nbsp;<a href="#speaker-yolanda_schaerli" class="alt">Yolanda Schaerli</a></li>
            <li>5:00 PM. &nbsp;&nbsp;&nbsp;<a href="#speaker-tal_danino" class="alt">Tal Danino</a></li>
            <li>6:00 PM. &nbsp;&nbsp;&nbsp;30min break</li>
            <li>6:30 PM. &nbsp;&nbsp;&nbsp;<a href="#speaker-igem_2020" class="alt">iGEM team &Eacute;vry Paris-Saclay 2020</a></li>
            </ul>
          </div>

        </section>


        <section class="page contentheight" id="page_local">

          <h1>Registration</h1>
          <div class="rightcol">
            <h1 class="alt">Local</h1>
            <p>
            Attendance to the workshop is free of charge.
            Please register via the <a href="http://www.disc-conference.org/wp/disc2020/registration/" class="alt">DISC'20 page</a>.
            Zoom links to talks will be provided on registration.
            </p>
          </div>

        </section>


        <!-- page -->
        <section class="page contentheight" id="page_speakers">
          <h1>Speakers</h1>
          <div class="rightcol">
            <h1 class="alt">Speakers</h1>
              <p>
              CELLS'20 speakers in order of appearance are:
              </p>
              <br>
      	    <!-- <p>Confirmed Speakers:</p> -->

              <hr id="speaker-jeff_hasty">
              <p>
              <a href="https://biodynamics.ucsd.edu/index.php/jeff/" class="alt2">
              Jeff Hasty (University of California, San Diego)
              </a><br>
	      <div class="title">Engineered Gene Circuits: From Clocks to Tumors and Small Ecologies: <a href="https://youtu.be/UeFMr3y-jQo">Video of the Talk on YouTube</a></div>
              </p>
              <p class="bio"><img src="images/jh.jpg" class="rightimg">
              Jeff Hasty received his Ph.D. in physics from the Georgia Institute of Technology in 1997, where he learned how to do science from his advisor Kurt Wiesenfeld. He was subsequently a postdoctoral fellow at Boston University, where he learned engineering from Jim Collins in the Applied BioDynamics Lab ('98-'01). Somewhere during his postdoctoral stay with Jim he mutated from a theoretical physicist into a hybrid computational/molecular biologist. He is currently at the University of California, San Diego, where he is a Professor in the Departments of Bioengineering and Molecular Biology, Director of the BioCircuits Institute, and Co-Director of the UC San Diego qBio Ph.D Specialization Program. He is considered a pioneer and leader in synthetic biology. His early career focused on tackling central impediments to the creation of an engineering discipline. He created genetic clocks to develop the underlying equations that drive circuit design, and used them to explore the coupling of gene circuits with their host genome. Faced with the inevitable noise of intracelluar gene regulation, he was the first to demonstrate how bacterial colonies can be engineered to function deterministically through the use of intercellular synchronization. He introduced the concept of “synergistic synchronization,” whereby two synchronization mechanisms couple colonies of bacteria at centimeter length scales. He used this concept to develop inexpensive biosensors that don't require complex optics. He has engineered periodic lysis of a bacterial colony such that population levels oscillate within a tumor microenvironment and release an encoded therapeutic. This design directly addresses the problem of systemic inflammatory response with programmed population control; since the colony is pruned after each oscillatory lysis event, the design mitigates an undesirable host response.
              </p>
              <p>
              <b>Abstract.</b>
	      For synthetic biology applications, intracellular variability is a major obstacle to the fidelity required for "programming" cells. A major theme in our research has been to investigate how determinism can arise from the synchronization of a large number of cells. I will review our basic approach to synchronization within the context of engineered bacteria that can be used to safely produce and deliver therapeutics from within solid tumors. Specifically, we have engineered a clinically relevant bacterium to lyse synchronously at a threshold population density and to release genetically encoded cargo. Following quorum lysis, a small number of surviving bacteria reseed the growing population, thus leading to pulsatile delivery cycles. We have extended this approach to small ecologies, whereby engineered periodic lysis can lead to ecological stability and generate in- teresting drug delivery schemes. Our current work explores how a small number of interacting species behave when constrained to grow on a surface as opposed to a well-mixed culture. It is known that non-hierarchical competitive dynam- ics, such as cyclical interactions, can sustain biodiversity. We have rationally designed a minimal microbial community with three strains of <i>E. coli</i> that cycli- cally interact through (i) the inhibition of protein production, (ii) the digestion of genomic DNA, and (iii) the disruption of the cell membrane. We find that intrinsic differences in these three major mechanisms of bacterial warfare lead to an unbalanced community that is, counterintuitively, dominated by the <i>weakest</i> strain. This work represents our first steps to probe the complexities of small engineered ecologies in environments that begin to approximate solid tumors.
	      </p>
              <br>


                            <hr id="speaker-david_doty">
              <p>
              <a href="https://web.cs.ucdavis.edu/~doty/" class="alt2">
              David Doty (University of California, Davis)
              </a><br>
              <div class="title">Crystals that think about how they’re growing: <a href="https://youtu.be/fUrwuQ9KLE0">Video of the Talk on YouTube</a></div>
              </p>
              <p class="bio"><img src="images/dd.jpg" class="rightimg">
               David Doty is an assistant professor of Computer Science at the University of California, Davis. He is broadly interested in problems at the intersection of physics, chemistry, biology, and computation. This does not mean the traditional "computation in service of natural science" (e.g., bioinformatics, computational chemistry, or molecular dynamics simulation). Rather, certain molecular systems — such as a test tube of reacting chemicals, a genetic regulatory network, or a growing crystal — can be interpreted as doing computation themselves... natural science in service of computation, if you will. He seeks to understand the fundamental logical and physical limits to computation by such means.
              </p>
              <p>
              <b>Abstract.</b>
               Advances in software engineering and programming languages have enabled vastly complex, yet reliable, electronic computing systems. Imagine a world where smart molecules, controlled by programmable chemical reactions, achieve the same level of precise automated control over the configuration of matter at the molecular level.
               </p>

        <p>
Biology offers inspiring examples of molecules that can store and process information to construct and control the sophisticated nanoscale devices that regulate the machinery of life. Yet biology offers almost no effective design principles for manufacturing such molecules ourselves. Much of synthetic biology relies on "alien technology": evolved proteins that, had evolution not furnished them, we would not know how to design ourselves.
        </p>

        <p>
DNA nanotechnology offers a different approach, enabling design of smart molecular systems from first principles. Theory that combines mathematical tiling and statistical-mechanical models of crystallization has shown that algorithmic behavior can be embedded within molecular self-assembly processes. Previous DNA nanotechnology results had experimentally demonstrated algorithmic "tile" self-assembly with up to 22 tile types, creating patterns such as Sierpinski triangles and binary counters. Despite that success, many information technologies exhibit a complexity threshold -- such as the minimum transistor count needed for a general-purpose computer -- beyond which there is a qualitative increase in the power of a reprogrammable system, and it has not been clear whether the biophysics of DNA self-assembly would allow that threshold to be exceeded.
        </p>

        <p>
Here we report the design and experimental validation of a DNA tile set containing 355 single-stranded tiles, reprogrammable by tile selection to implement a wide variety of 6-bit algorithms, including copying, sorting, recognizing palindromes and multiples of 3, random walking, obtaining an unbiased choice from a biased random source, electing a leader, simulating Turing-universal cellular automata, generating deterministic and randomized patterns, and serving as a period 63 counter. The system is quite reliable: averaged across the 21 implemented circuits, the per-tile error rate is less than 1 in 3000.
        </p>

        <p>
The breadth, complexity, and reliability of this system demonstrates that fabrication of complex, atomically precise structures, using rationally-designed, bottom-up molecular programming, is a feasible long-term goal. Nanoscience need no longer take inspiration only from biology; it can take inspiration from computer science.
        </p>

        <p>
        <b>paper</b>: <a href="https://www.nature.com/articles/s41586-019-1014-9">https://www.nature.com/articles/s41586-019-1014-9</a>
        <br />
        additional links related to paper: <a href="http://web.cs.ucdavis.edu/~doty/papers/#drmaurdsa">http://web.cs.ucdavis.edu/~doty/papers/#drmaurdsa</a>
        </p>
              </p>
              <br>


              <hr id="speaker-kurt_mehlhorn">
              <p>
              <a href="https://people.mpi-inf.mpg.de/~mehlhorn/" class="alt2">
              Kurt Mehlhorn (Max Planck Institute for Informatics)
              </a><br>
              <div class="title">Physarum Computations: <a href="https://youtu.be/qMNrBZSRuw0">Video of the Talk on YouTube</a></div>
              </p>
              <p class="bio"><img src="images/km.jpg" class="rightimg">
              Kurt Mehlhorn is director emeritus at the Max Planck Institute for Computer Science and Senior Professor at the Saarland University. He works on algorithms in a broad sense, is the author of several books and more than 200 research papers. Many of his former students and postdocs are now professors. He received a number of awards, among them the ACM Theory and Practice Award and the Erasmus Medal of the Academia Europeae. He is co-founder of Algorithmic Solutions GmbH. 
              </p>
              <p>
              <b>Abstract.</b>
              Physarum is a slime mold that can apparently solve shortest path problems and build elegant and efficient networks. We report on recent work on understanding the network building capability and on the influence of noise on the network dynamics.
              </p>
              <br>


              <hr id="speaker-tom_de_greef">
              <p>
              <a href="https://www.tue.nl/en/research/researchers/tom-de-greef/" class="alt2">
              Tom de Greef (Eindhoven University of Technology)
              </a><br>
              <div class="title">Programmable DNA-based Communication in Populations of Synthetic Cells: <a href="https://youtu.be/kNDxrFMakTQ">Video of the Talk on YouTube</a></div>
        </p>
              <p class="bio"><img src="images/tdg.jpg" class="rightimg">
        Tom de Greef studied Biomedical Engineering at Eindhoven University of Technology (TU/e, the Netherlands), where he received his MSc degree cum laude in 2004. He then started his PhD research at the TU/e department of Chemical Engineering and Chemistry, where he graduated in 2008 on a thesis on novel polymeric materials based on quadruple hydrogen-bonding motifs, supervised by professors E. W. (Bert) Meijer and Rint Sijbesma. He subsequently moved to the Computational Biology group at the TU/e department of Biomedical Engineering (TU/e) headed by Prof. Peter Hilbers, for postdoctoral research on self-assembling systems from a computational perspective. In 2010, he became assistant professor and in 2016 he was promoted to associate professor. In 2013, Tom de Greef was a visiting scholar in the group of Prof. David Weitz at Harvard University (Cambridge, USA), working on protein affinity screening using droplet microfluidics. In 2012 and 2013 the Netherlands Organization of Scientific Research (NWO) awarded him a VENI grant and an ECHO-STIP grant, respectively. In 2015 he received an ERC starting Grant and in 2016 an NWO VIDI grant. Tom de Greef is a core member of the Institute for Complex Molecular Systems (ICMS), a core member of the Dutch Gravitation program "Functional Molecular Systems" and a junior faculty member of the Gravitation program "Materials-Driven Regeneration". He received the 2017 Cram Lehn Pedersen prize in supramolecular chemistry, and in 2018 he was awarded a Microsoft PhD scholarship for the development of a DNA computer.
              </p>
              <p>
              <b>Abstract.</b>
        The development of highly orthogonal molecular communication channels is a crucial step towards engineering artificial cell-scale systems. Here, we present a general and scalable platform entitled 'Biomolecular Implementation of Protocellular Communication' (BIO-PC) to engineer multichannel molecular communication networks between populations of non-lipid microcapsules. Our method leverages the modularity and scalability of enzyme-free DNA strand-displacement circuits to develop protocellular consortia that can sense, process and respond to DNA-based messages. We engineer a rich variety of biochemical communication devices capable of cascaded amplification, bidirectional communication, sender-receiver functions and distributed computational operations. Encapsulation of DNA strand-displacement circuits allows their use in concentrated serum where non-compartmentalized DNA circuits cannot operate. BIO-PC enables reliable execution of distributed DNA-based molecular programs in biologically relevant environments and opens new directions in DNA computing and minimal cell technology.
              </p>
              <br>


              <hr id="speaker-yolanda_schaerli">
              <p>
              <a href="https://www.yschaerli.com/" class="alt2">
              Yolanda Schaerli (University of Lausanne)
              </a><br>
              <div class="title">Synthetic gene regulatory networks for spatiotemporal pattern formation: <a href="https://youtu.be/A8sv6ybbcNY">Video of the Talk on YouTube</a></div>
              </p>
              <p class="bio"><img src="images/ys.jpg" class="rightimg">
             Yolanda Schaerli is an Assistant Professor of Synthetic Biology at the Department of Fundamental Microbiology, University of Lausanne, Switzerland. Research in her laboratory uses a bottom-up synthetic biology approach to understand the mechanisms, properties, and evolution of gene regulatory networks, especially those involved in pattern formation.
              </p>
              <p>
              <b>Abstract.</b>
             Synthetic biology has emerged as a multidisciplinary field that provides new tools and approaches to address longstanding problems in biology. It integrates knowledge from biology, engineering and mathematics to build – rather than to simply observe and perturb – biological systems that emulate natural counterparts or that explore alternative solutions beyond nature’s repertoire.
             </p>
        
              <p>
              In my group we use this bottom-up synthetic biology approach to understand the mechanisms, properties and evolution of gene regulatory networks, with a focus on networks involved in spatiotemporal pattern formation.
              </p>

              <p>
              In my talk, I will present our latest research on pattern-forming synthetic networks in populations of E. coli cells, including our work on the toggle switch and on multistable and dynamic synthetic circuits using CRISPR interference for precise temporal and spatial control of gene expression.
              </p>
              <br>
              

              <hr id="speaker-tal_danino">
              <p>
              <a href="http://daninolab.nyc" class="alt2">
              Tal Danino (Columbia University)
              </a><br>
              <div class="title">Engineering bacteria as a cancer therapy: <a href="https://youtu.be/lKH5f0CyM4U">Video of the Talk on YouTube</a></div>
              </p>
              <p class="bio"><img src="images/td.jpg" class="rightimg">
              Tal Danino is an Assistant Professor in the Department of Biomedical Engineering at Columbia University. His lab focuses on engineering bacteria for biomedical applications, with a particular emphasis on developing bacteria as a cancer therapy. Originally from Los Angeles, Tal received a Ph.D. in Bioengineering from UCSD in Jeff Hasty's lab, and was a postdoctoral fellow at the Koch Institute for Integrative Cancer Research with Sangeeta Bhatia. He is the recipient of awards including the NSF CAREER Award, Era of Hope Scholar Award, TED Fellow, and NIH Pathway to Independence Award. He directs the Synthetic Biological Systems Laboratory and is a member of the Herbert Irving Comprehensive Cancer Center and Data Science Institute.
              </p>
              <p>
              <b>Abstract.</b>
              Over the past decade, microbiome research has revealed an astounding prevalence of bacteria in both healthy and diseased tissue. This natural presence of bacteria has renewed interest in programming microbes to sense and respond to disease states using approaches from synthetic biology. One particular focus is cancer therapy, where a multitude of studies have demonstrated selective colonization of solid tumors by bacteria, primarily due to reduced immune surveillance in tumor cores. In this talk, I will describe our laboratory's progress in building a multi-scale framework for engineering bacteria as a diagnostic and therapeutic agent for cancer. We use a methodology that bridges in silico computational modeling, in vitro characterization and platform development, and in vivo mouse models for cancer. We will highlight recent examples of bacteria programmed to sense and respond to tumor environments and release specific therapeutic payloads ranging from cytotoxic to immunomodulatory agents. This approach allows for the creation of engineered systems that intelligently sense and respond to diverse environments, ultimately adding specificity and efficacy that extends beyond the capabilities of molecular-based therapeutics. 
              </p>
              <br>


              <hr id="speaker-igem_2020">
              <p>
              <a href="https://2020.igem.org/Team:Evry_Paris-Saclay" class="alt2">
              iGEM Team &Eacute;vry Paris-Saclay 2020
              </a><br>
              <div class="title">How to train your microbe for Rosewood sensing? <a href="https://youtu.be/8XNMIUEZIbc">Video of the Talk on YouTube</a></div>
              </p>
              <p class="bio"><img src="images/igem.png" class="rightimg" style="height: 100px;">
	             The International Genetically Engineered Machine (iGEM) is an international competition where student teams develop synthetic biology projects over each summer and then present their research at the Giant Jamboree in October/ November. The team of University of &Eacute;vry Paris-Saclay has been successfully competing at iGEM since 2012, addressing many social and scientific challenges through their projects, and winning medals and prizes. This year, their project tackles the problem of illegal wildlife trafficking and trade.
              </p>
              <p>
              <b>Abstract.</b>
              The Rosewood species is the source of the most trafficked wildlife products worldwide. We are developing cheap, portable and easy-to-use biosensors that will accelerate the identification process and thus empower local authorities to identify and stop the logging of this species. Our biosensor uses engineered molecular machinery of the common gut bacterium (E. coli) to sense nucleic acid signatures specific to the Rosewood tree. We will demonstrate how to go from the design to the final application of the biosensors, identifying the trafficked Rosewood to the family, phylum, or the species level.
              </p>
              <br>

      	  </div>
	       </section>
	

        <!-- page -->
        <section class="page contentheight" id="page_commitee">

          <h1>Committee</h1>
          <div class="rightcol">
            <h1 class="alt">Committee</h1>
            <p>
            <a href="http://www.lsv.fr/~mfuegger/" class="alt">Matthias F&uuml;gger</a>,
            CNRS & LSV, ENS Paris-Saclay, Universit&eacute; Paris-Saclay & Inria
            </p>
            <p>
            <a href="https://www.micalis.fr/micalis_eng/Poles-and-teams/Pole-Systems-and-Synthetic-Microbiology/Metabolic-Engineering-by-Retro-Synthesis-Jean-Loup-Faulon/Team-members/Manish-Kushwaha" class="alt">Manish Kushwaha</a>, Micalis Institute, INRAE-AgroParisTech, Universit&eacute; Paris-Saclay
            </p>
            <p>
            <a href="https://www.lri.fr/~nowak/" class="alt">Thomas Nowak</a>,
            Universit&eacute; Paris-Saclay<br><br>
            </p>
          </div>

        </section>

        <!-- page -->
        <section class="page contentheight" id="page_sponsors">

          <h1>Sponsors</h1>
          <div class="rightcol">
            <h1 class="alt">Sponsors</h1>
            
            <p>
            <a href="https://digicosme.lri.fr/tiki-index.php">
              <img src="logos/logo-DigiSaclaysmall.png" width="150"><br>
            </a> DigiCosme (working group <a href="https://parsys.lri.fr/HicDiesMeus/">HicDiesMeus</a>)
            </p>

            <br>
            <br>
            <br>

          </div>

        </section>

        <!-- page -->
        <section class="page contentheight" id="page_previous">

          <h1>Previous<br>workshops</h1>
          <div class="rightcol">
            <h1 class="alt">Previous workshops</h1>
            
            <p>
            <a href="https://parsys.lri.fr/CELLS/2019.html" class="alt">
            CELLS'19
            </a>
            </p>

            <br>
            <br>
            <br>

          </div>

        </section>


      </section>


    </center>
      
    <!-- Scripts -->
    <script src="assets/js/jquery.min.js"></script>
    <script src="assets/js/skel.js"></script>
    <script src="assets/js/skel-layout.js"></script>
    <script src="assets/js/util.js"></script>
    <script src="assets/js/main.js"></script>
	

    </script>
    
  </body>
</html>
