<!DOCTYPE HTML>
<html>
  <head>
    <title>CELLS Workshop</title>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
    <link rel="stylesheet" href="assets/css/main.css?v=4" />
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.9/css/all.css" integrity="sha384-5SOiIsAziJl6AWe0HWRKTXlfcSHKmYV4RBF18PPJ173Kzn7jzMyFuTtk8JA7QQG1" crossorigin="anonymous">
    <link href="https://fonts.googleapis.com/css?family=Roboto:300,300italic,400,400italic,500,500italic,600,600italic,700,700italic,900,900italic" rel="stylesheet">
    <style>
      .mySlides {
        display:none;
      }
    </style>
  </head>

  <body>
    <center>

      <!-- Header -->
      <header id="header">

      	<nav id="nav">
      	  <ul>
            <li><a href="#page_disc">DISC'21</a></li>
      	    <li><a href="#page_scope">Scope</a></li>
      	    <li><a href="#page_program">Program</a></li>
            <li><a href="#page_local">Local</a></li>
            <li><a href="#page_speakers">Speakers</a></li>
      	    <li><a href="#page_commitee">Committee</a></li>
            <li><a href="#page_sponsors">Sponsors</a></li>
            <li><a href="#page_previous">Previous workshops</a></li>
            <li><a href="2020.html">CELLS'20</a></li>
            <li><a href="2019.html">CELLS'19</a></li>
      	  </ul>
      	</nav>

      </header>

      <!-- Toggle Menu -->
      <a href="#menu" class="navPanelToggle">
      	<img src="logos/menu.svg"/>
      </a>
	
      <!-- Main part --> 
      <section id="main">
        
        <!-- main page -->
        <section class="page contentheight" id="first_page">
        

          <div class="bannerdiv">
           <img src="images/cells.svg" class="bannerimage">
          <span class="caption">CELLS: Workshop on Computing among Cells</span>
          <!--span class="caption2">October 15th and 16th, 2020 (online)<br><br></span>
          <a href="https://forms.gle/6WQeowDx46yH37Hi7" class="alt-reg">Register here</a><br><br><br>
          <a href="CELLS.ics" class="alt-button">Add events to calendar</a-->
          <!--
          <span class="caption2">We would like to thank all speakers and participants of the CELLS'20 workshop! For videos please see the <a href="2020.html" class="alt">CELLS'20</a> page
          <br>Looking forward to see you in 2021.<br><br></span>
          -->
          </div>

        </section>

        <section class="page contentheight" id="page_disc">

          <h1>DISC'21</h1>
          <div class="rightcol">
            <h1 class="alt">DISC'21</h1>
            <p>
            CELLS is a DISC 2021 workshop. Please see <a href="http://www.disc-conference.org/wp/disc2021/" class="alt">the DISC page</a> for further information.
            The workshop will take place online.
            <!--Attendance to the workshop is free of charge.-->
            Zoom links to the talks are provided after registration.
            </p>
            <!--b>Free registration:</b>
            <p><a href="https://forms.gle/6WQeowDx46yH37Hi7" class="alt">Registration</a> is now open via the DISC 2020 page.
            </p-->
          </div>

        </section>

        <!-- page 2 -->
        <section class="page contentheight" id="page_scope">

          <h1>Scope</h1>
          <div class="rightcol">
          	<h1 class="alt">Scope</h1>
            <p>
            The workshop is intended to cover topics both from computer science and microbiology with a focus on computation emerging from microbiological units (bacteria, viruses, DNA, etc.) interacting with each other.
            </p>
            <p>
            Topics include, but are not limited to: synthetic biology, bacterial computing, DNA computing, population protocols, Markov chains, modeling with cellular automata, infection models, and evolutionary models.
            </p>
          </div>

        </section>

        <section class="page contentheight" id="page_program">

          <h1>Program</h1>
          <div class="rightcol">
            <h1 class="alt">Program</h1>
            <p>
            Talks will be held online on October 8th, 2021 (starting 11 AM, Central European Summer Time).
            </p>

            <p>
            Friday, October 8, 2021 (CEST times):
            </p>
            <ul>
            <li>11:00 AM. &nbsp;<a href="#speaker-luca cardelli" class="alt">Luca Cardelli</a></li>
            <li>12:00 PM. &nbsp;<a href="#speaker-kiran raosaheb patil" class="alt">Kiran Raosaheb Patil</a></li>
            <li>1:00 PM. &nbsp;&nbsp;&nbsp;60min break</li>
            <li>2:00 PM. &nbsp;&nbsp;&nbsp;<a href="#speaker-mustafa khammash" class="alt">Mustafa Khammash</a></li>
            <li>3:00 PM. &nbsp;&nbsp;&nbsp;<a href="#speaker-angel goni-moreno" class="alt">Ángel Goñi-Moreno</a></li>
            <li>4:00 PM. &nbsp;&nbsp;&nbsp;<a href="#speaker-nancy lynch" class="alt">Nancy Lynch</a></li>
            </ul>
            </p>
          </div>

        </section>


        <section class="page contentheight" id="page_local">

          <h1>Registration</h1>
          <div class="rightcol">
            <h1 class="alt">Local</h1>
            <p>
            Attendance to the workshop is free of charge.
            Please register via the <a href="http://www.disc-conference.org/wp/disc2021/registration/" class="alt">DISC'21 page</a>.
            Zoom links to talks will be provided on registration.
            </p>
          </div>

        </section>



        <!-- page -->
        <section class="page contentheight" id="page_speakers">
          <h1>Speakers</h1>
          <div class="rightcol">
            <h1 class="alt">Speakers</h1>
              <p>
              CELLS'21 speakers in order of appearance are:
              </p>

              <hr id="speaker-luca cardelli">
              <p>
              <a href="http://lucacardelli.name/index.html" class="alt2">
              Luca Cardelli (University of Oxford)
              </a><br>
              <div class="title">Sequenceable Event Recorders </div>
              </p>
              <p class="bio"><img src="images/lc_21.png" class="rightimg">
        Luca Cardelli has an M.Sc. in computer science from the University of Pisa, and a Ph.D. in computer science from the University of Edinburgh. He worked at Bell Labs Murray Hill from 1982 to 1985, at Digital Equipment Corporation Systems Research Center in Palo Alto from 1985 to 1997, and at Microsoft Research in Cambridge UK from 1997 to 2018 where he was head of the Programming Principles and Tools and Security groups until 2012. Since 2013 he is a Royal Society Research Professor at the University of Oxford.
              </p>
        <p>
        His main interests are in programming languages and concurrency, and more recently in programmable biology and nanotechnology. He is a Fellow of the Royal Society, a Fellow of the Association for Computing Machinery, an Elected Member of the Academia Europaea, and an Elected Member of the Association Internationale pour les Technologies Objets.
        </p>
              <p>
              <b>Abstract.</b>
        With recent high-throughput technology we can synthesize large heterogeneous collections of DNA structures, and also read them all out precisely in a single procedure. Can we use these tools, not only to do things faster, but also to devise new techniques and algorithms? In this paper we examine some DNA algorithms that assume high-throughput synthesis and sequencing. We record the order in which N events occur, using N^2 redundant detectors but only N distinct DNA domains, and (after sequencing) reconstruct the order by transitive reduction.  
               </p>
              <br>   



    <hr id="speaker-kiran raosaheb patil">
              <p>
              <a href="https://scholar.google.pt/citations?user=RDAe92sAAAAJ&hl=en" class="alt2">
              Kiran Raosaheb Patil (University of Cambridge)
              </a><br>
              <div class="title">Cooperation and competition in microbial communities - from earth microbiome to synthetic communities</div>
              </p>
              <p class="bio"><img src="images/krp_21.png" class="rightimg">
        Kiran studied Chemical Engineering at the Indian Institute of Technology (Mumbai, India). He moved to the Technical University of Denmark (DTU) to work with Prof. Jens Nielsen and obtained his PhD in Systems Biology. Kiran was then appointed as Assistant Professor at DTU where he worked on transcriptional regulation and metabolic engineering. In 2010, Kiran joined the Structural and Computational Biology Unit at the European Molecular Biology Laboratory (EMBL-Heidelberg, Germany). He was appointed Director of Research at the MRC Toxicology Unit (University of Cambridge) in 2019. The Patil lab has developed novel tools and model systems to decipher metabolic interactions in complex microbial communities.

              </p>
              <p>
              <b>Abstract.</b>
        Our planet is, in many ways, a microbial planet. More than trillion microbial species are estimated to habitat Earth, with most living in multi-species communities. These communities occupy almost every habitable corner, from deep ocean floor to our food and our gut. Metagenomic studies are providing an excellent overview of this fascinating diversity, while numerous functional studies have demonstrated the key role these microbes play in the health of the people and the planet. I will present my lab’s efforts towards gaining mechanistic insights into the organisational principles of microbial communities using genome-scale metabolic models. In particular, I will discuss how the tug-of-war between competition and cooperation has shaped microbiomes across habitats and present our efforts to use these insights in designing synthetic communities.
               </p>
              <br>   


   <hr id="speaker-mustafa khammash">
              <p>
              <a href="https://bsse.ethz.ch/ctsb" class="alt2">
              Mustafa Khammash (ETH Zurich)
              </a><br>
              <div class="title">Genetic Proportional-Integral-Derivative Control Systems</div>
              </p>
              <p class="bio"><img src="images/km_21.jpg" class="rightimg">
              Mustafa Khammash is the Professor of Control Theory and Systems Biology in the Department of Biosystems Science and Engineering (D-BSSE) at ETH-Zurich. After receiving his Ph.D. in Electrical Engineering from Rice University in 1990, he joined the Electrical Engineering Department at Iowa State University (ISU). While at ISU, he created the Dynamics and Control Program and led that control group until 2002, when he joined the faculty of the University of California, Santa Barbara. In Santa Barbara, he served as the Director of the Center for Control, Dynamical Systems and Computation from 2005 to 2011. In 2011 he moved with his group to Switzerland, joining the Department of Biosystems Science and Engineering at ETH Zurich. At ETH, he was department chair (2015-2017). </p>
              <p>
              Khammash is a Fellow of the IEEE (2007), the International Federation of Automatic Control (2010), and the Japan Society for the Promotion of Science (2003). He is a winner of the ISU Early achievement in Research and Scholarship Award, the Young Engineering Research Faculty Award, the ETH Golden Owl Award, and the European Research Council Advanced Grant. 
              </p>
              <p>
              Dr. Khammash works at the interface of the areas of control theory, systems biology, and synthetic biology. He has pioneered the field of Cybergenetics where he has been developing novel theory, computational, and experimental methodologies for controlling living cells using both computer and genetic control systems.
              </p>
              <p>
              <b>Abstract.</b>
        Proportional-Integral-Derivative (PID) feedback controllers have been the most widely used controllers in industry for almost a century due to their good performance, simplicity, and ease of tuning. Motivated by their success in various engineering disciplines, PID controllers recently found their way into synthetic biology, where the design of feedback molecular control systems has been identified as an important goal. In this talk, I consider the realization of PID controllers via biomolecular interactions. I will propose a hierarchy of new biochemical reaction network topologies that realize biomolecular PID controllers. I will explore the different performance features of these controllers and will discuss their practical implementation in living cells. This family of new controllers serves as a starting point for the design and construction of advanced high-performance biomolecular control systems that robustly regulate the transient and steady-state dynamics of living cells.
               </p>
              <br>   

  <hr id="speaker-angel goni-moreno">
    <p>
              <a href="https://sites.google.com/site/angelgonimoreno/" class="alt2">
              Ángel Goñi-Moreno (Universidad Politécnica de Madrid)
              </a><br>
              <div class="title">Genetic circuits beyond genes: contextual dependencies as design parameters</div>
              </p>
              <p class="bio"><img src="images/agm_21.jpeg" class="rightimg">
              Ángel Goñi-Moreno is the head of the Biocomputation Laboratory at the Centre for Plant Biotechnology and Genomics (CBGP), in Madrid. He studied Computer Engineering before received his Ph.D. in Synthetic Biology from the Technical University of Madrid (UPM) in 2010, from which he also gained an MSc. degree in Artificial Intelligence. He then joined Martyn Amos' group at Manchester Metropolitan University (UK) as a postdoc, working in mathematical modelling and computational synthetic biology, with close collaborations in experimental laboratories in Europe. In early 2013, he came back to Madrid to join Victor de Lorenzo's lab which is part of the Systems Biology Program at CNB-CSIC (Centro Nacional de Biotecnología - CSIC). During this period, he worked in all areas of synthetic biology, from building models to carrying out wet-lab experiments, which was an excellent interdisciplinary training. In 2016 he moved to Newcastle University (UK) to establish his own independent team within the Interdisciplinary Computing and Complex Biosystems (ICOS) research group. At Newcastle he led a team carrying out research at the interface of computing and biology, formed by physicists, engineers, biochemists, biologists and computer scientists. In September 2020 he returned to the Technical University of Madrid to establish the Biocomputation Lab, which is the first hybrid theoretical and experimental laboratory on biocomputation and unconventional computing of this institution.
              </p>
              <p>
              <b>Abstract.</b>
        The design and implementation of genetic circuits is a major sub-field within synthetic biology. Genetic components are arranged generating pre-defined regulatory cascades where some inputs are turned into outputs. This allows us to build biological computations, for instance, in the form of Boolean functions where the combination of inputs turn ON or OFF a specific reporter protein. Although this conceptual mapping has been relatively successful, there are stability and robustness issues that cannot be overlooked. We argue that the performance of biological circuits in living cells cannot be reduced to DNA sequences. While this statement is rather obvious, there are still little efforts on characterising the mechanistic complexity of cellular-based networks for the sake of a robust bottom-up design. The term contextual dependencies refer to all those features of the cellular background that impact on the performance of genes. A specific biological circuit may be re-programmed (this is, we can change its performance) by modifying its contextual dependencies, and not its DNA sequence. The genetic components are the same. Intracellular space, for instance, is a useful parameter that had not been exploited before for biodesign; the same genetic components but at different relative positions within the chromosome will alter their performance. There are more contextual parameters, including plasmid vectors, host chassis or the environmental conditions. All of them can be designed in order to obtain different performances out of the same genetic circuit. By moving away from reductionism and embracing complexity, we may also find novel models of bio-computation that are more powerful than Boolean logic. 
               </p>
              <br>  


    <hr id="speaker-nancy lynch">
              <p>
              <a href="https://people.csail.mit.edu/lynch/" class="alt2">
              Nancy Lynch (MIT)
              </a><br>
              <div class="title">An Algorithmic Theory of Brain Behavior</div>
              </p>
              <p class="bio"><img src="images/nl.jpg" class="rightimg">
        Nancy Lynch is the NEC Professor of Software Science and Engineering in MIT's EECS department.  She heads the Theory of Distributed Systems research group in the MIT Computer Science and Artificial Intelligence Laboratory. She received her PhD from MIT and her B.S. from Brooklyn College, both in Mathematics.
              </p>
        <p>
        Lynch has (co-)written many research articles about distributed algorithms and impossibility results, and about formal modeling and verification of distributed systems. Her best-known contribution is the ``FLP'' impossibility result for reaching consensus in asynchronous distributed systems in the presence of failures, with Fischer and Paterson, followed by results with Dwork and Stockmeyer on algorithms for reaching consensus under restricted failure assumptions. Other contributions include the I/O automata system modeling frameworks, with Tuttle, Kaynar, Segala, and Vaandrager. Lynch is the author of the textbook ``Distributed Algorithms'' and a co-author of ``The Theory of Timed I/O Automata'' and "Atomic Transactions". She is an ACM Fellow, a member of the National Academy of Sciences and the National Academy of Engineering, and a Fellow of the American Academy of Arts and Sciences. She has been awarded the Dijkstra Prize (twice), the van Wijngaarden prize, the Knuth prize, the Piore award, the Athena Lecturer award, and an IEEE Technical Achievement award for distributed computing. She has supervised approximately 100 PhD students, Masters students, and postdocs.
        </p>
        <p>
        Lynch is interested in all aspects of distributed computing theory, including modeling, algorithm design, analysis, lower bounds, and applications. Currently, she is especially interested in algorithms for ``difficult'' platforms, which are subject to noise, failures, and changes. Her recent work has focused on biological distributed algorithms (insect colonies and brains).
        </p>
              <p>
              <b>Abstract.</b>
        This talk will describe a recent and current research project being carried out by my group.  This project aims at understanding computation in the brain, in terms of abstract, interacting distributed algorithms.  It assumes a computational model based on weighted directed graphs of stochastic computing elements corresponding to neurons, operating synchronously in parallel.  It studies problems that are typical of those solved by brains:  problems of focus and attention, decision-making, similarity detection and clustering, visual and auditory recognition, and representation and learning of concepts. The project uses techniques from theoretical computer science (especially, distributed algorithms), while taking inspiration from experimental neuroscience.  This work has both biological and computer science motivations:  Its algorithmic perspective should help in understanding computational mechanisms employed by biological neural networks and the limitations of such networks.  On the other hand, many problems solved by these networks are also fundamental in computer science.  Studying them in the setting of biological neural networks should offer a new perspective and yield new theoretical results.  Biological algorithms are naturally robust, self-stabilizing, and adaptive to changes—properties that are important for modern computer systems.
               </p>

        <p>
  Specifically, in recent years, we have been exploring aspects of theory for brain computation, based on synchronous, stochastic, Spiking Neural Networks (SNNs).  Inspired by tasks performed by real brains, we have defined abstract problems to be solved by SNNs, such as the Winner-Take-All and similarity-detection problems, analyzed them in terms of costs such as network size and convergence time, and proved corresponding lower bounds. We have also collaborated with experimental neuroscientists in modeling specific brain mechanisms.  Now we are trying to pull these ideas together into a coherent theory, by focusing on the central issues of how concepts are represented in the brain, how those representations are used, and how they may be learned. “Concepts” here include logical concepts, such as structured and unstructured data and linguistic constructs, and physical concepts, such as moving objects.
        </p>
              </p>
              <br>
              

      	  </div>
	       </section>
	

        <!-- page -->
        <section class="page contentheight" id="page_commitee">

          <h1>Committee</h1>
          <div class="rightcol">
            <h1 class="alt">Committee</h1>
            <p>
            <a href="http://www.lsv.fr/~mfuegger/" class="alt">Matthias F&uuml;gger</a>,
            CNRS & LMF, ENS Paris-Saclay, Universit&eacute; Paris-Saclay & Inria
            </p>
            <p>
            <a href="https://www.micalis.fr/micalis_eng/Poles-and-teams/Pole-Systems-and-Synthetic-Microbiology/Metabolic-Engineering-by-Retro-Synthesis-Jean-Loup-Faulon/Team-members/Manish-Kushwaha" class="alt">Manish Kushwaha</a>, Micalis Institute, INRAE-AgroParisTech, Universit&eacute; Paris-Saclay
            </p>
            <p>
            <a href="https://www.lri.fr/~nowak/" class="alt">Thomas Nowak</a>,
            Universit&eacute; Paris-Saclay<br><br>
            </p>
          </div>

        </section>

        <!-- page -->
        <section class="page contentheight" id="page_sponsors">

          <h1>Sponsors</h1>
          <div class="rightcol">
            <h1 class="alt">Sponsors</h1>
            
            <p>
            <a href="https://anr.fr">
              <img src="logos/ANR-logo-2021-complet.jpg" width="250"><br>
            </a> ANR project <a href="https://dreamy.run">DREAMY</a>
            </p>
            <br>

            <p>
            <a href="https://digicosme.lri.fr/tiki-index.php">
              <img src="logos/logo-DigiSaclaysmall.png" width="150"><br>
            </a> DigiCosme (working group <a href="https://parsys.lri.fr/HicDiesMeus/">HicDiesMeus</a>)
            </p>

            <br>
            <br>
            <br>

          </div>

        </section>

        <!-- page -->
        <section class="page contentheight" id="page_previous">

          <h1>Previous<br>workshops</h1>
          <div class="rightcol">
            <h1 class="alt">Previous workshops</h1>
            
            <p>
            <a href="2020.html" class="alt">
            CELLS'20
            </a>
            </p>

            <p>
            <a href="2019.html" class="alt">
            CELLS'19
            </a>
            </p>

            <br>
            <br>
            <br>

          </div>

        </section>


      </section>


    </center>
      
    <!-- Scripts -->
    <script src="assets/js/jquery.min.js"></script>
    <script src="assets/js/skel.js"></script>
    <script src="assets/js/skel-layout.js"></script>
    <script src="assets/js/util.js"></script>
    <script src="assets/js/main.js"></script>
	

    </script>
    
  </body>
</html>
